src/d3a/d3a_core/simulation.py

```diff
@@ -313,7 +313,7 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
         self.simulation_config.external_redis_communicator.start_communication()
         self._update_and_send_results()
 
-        if d3a.constants.IS_CANARY_NETWORK is True:
+        if RUN_IN_REALTIME:
```

Comment:
That's ok, but keep in mind that not accessing the constant with the full path will have the effect that you cannot override its value in tests. If that's okay then no problem at all

Reply:
ah ok, not, will use the `d3a.constants. RUN_IN_REALTIME ` then,

Reply:
will be fixed when merging D3ASIM-3004_fix_cn_run.